digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -1,15 +1,57 @@
 <template>
   <div>
     <div v-if="!isLoaded" class="main-menu">
-      <h1>Digital Deck</h1>
+      <h1 class="heading">Digital Deck</h1>
+      <button class="button" v-on:click="toggleForm1()">Create</button>
+      <button class="button" v-on:click="toggleForm2()">Join</button>
       <br />
-      <span>Decks</span><br />
-      <input v-model.number="deckSelected" type="number" min="1" oninput="validity.valid||(value='');"/><br />
-      <span>Cards on hand</span><br />
-      <input v-model.number="cardsPerPlayer" type="number"  min="1" oninput="validity.valid||(value='');"/><br />
-      <span>Cards on table</span><br />
-      <input v-model.number="cardsOnTable" type="number"  min="0" oninput="validity.valid||(value='');"/><br /><br />
-      <button @click="submitForm" style="cursor: pointer;">Start</button>
+      <div v-if="form1" class="input-form">
+        <form>
+          <div class="number-box">
```

Comment:
Can you please also add another field so players can input their name and store this data in the backend as well? We'll need that for player display

Reply:
That is gonna required a lot of change in the backend. It will be a big change as some of our tests will also need to be changed. And if you look in `CardDistService`, I don't see how we can get the names there without changing a lot of things. I think we can do without names for now, as it is not really a core feature.

Reply:
The way I see it, a lot of changes have to be made to `CardDistService` regardless, since I did point out early in our first sprint that the current code will not work for multiplayer. Aside from that, I don't think adding names will be that much work, since it's just 1 more instance variable in `Player.js`. I also think it's a core part of acceptance test since I don't think our app would make sense if none of the players can tell who is who.